rgislackbot/repo_command/github_api/urls.py

```diff
@@ -0,0 +1,5 @@
+
+class Urls(object):
+    def __init__(self):
+        self.base = 'https://api.github.com/repos/raginggeek/RGISlackBot/'
+        self.pull_requests = self.base + 'pulls/'
```

Comment:
PEP8 standards says new lines at the ending of every file.(also important for linux/unix compatibility.